frontend/src/libs/components/button/button.tsx

```diff
@@ -3,10 +3,17 @@ import styles from './styles.module.scss';
 type Properties = {
   label: string;
   type?: 'button' | 'submit';
+  style?: 'primary' | 'secondary';
+  disabled?: boolean;
```

Comment:
i would suggest to change disabled to isDisabled because of quality criteria: B19. Boolean variables start with a prefix that can be answered with "yes".

Reply:
You need to follow quality criteria 😉